docs/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md

```diff
@@ -40,15 +45,15 @@ The `connection.connection` object found in `./config/database.js` (or `./config
 
 | Parameter  | Description                                                                                                                   | Type                  |
 |------------|-------------------------------------------------------------------------------------------------------------------------------|-----------------------|
-| `connectionString`| Database connection string. When set, it overrides the other `connection.connection` properties. To disable use an empty string: `''`.                    | `String`                  |
+| `connectionString`| Database connection string. When set, it overrides the other `connection.connection` properties. To disable use an empty string: `''`. Available in v4.6.2+                    | `String`                  |
```

Comment:
I think we could use additional formatting to make the "Available in […]" stand out.
For consistency, I would also suggest we use the same formatting everywhere in the table (see line 55, "Used only […]")
Adding a line break and using italics should be enough. What do you think?

Reply:
OK with me. I used bold since that is what was on line 55.